actions/app.go

```diff
@@ -38,8 +44,21 @@ func App() *buffalo.App {
 			app.Use(middleware.ParameterLogger)
 		}
 
-		app.GET("/", HomeHandler)
+		// middleware
+		app.Use(authenticateRequest)
+
+		authGroup := app.Group("/auth")
+		authGroup.Middleware.Skip(authenticateRequest, createUserHandler, loginHandler) // do not verify a token for these registration/login handlers
+		authGroup.POST("/user", createUserHandler)
+		authGroup.POST("/login", loginHandler)
+
+		app.GET("/user/{userID}", userDataHandler)
+		app.PUT("/user/monster", renameMonsterHandler)
+		app.POST("/user/monster/{monsterNo}", addMonsterHandler)
+		app.POST("/user/monster/{monsterNo}/attack/{attackName}", addMonsterAttackHandler)
```

Comment:
Should these be a group?

Reply:
Yeah it would be cool to nest them in a "/user" group